staging/src/k8s.io/apimachinery/pkg/test/runtime_serializer_protobuf_protobuf_test.go

```diff
@@ -218,7 +218,7 @@ func TestProtobufDecode(t *testing.T) {
 		0x0a, 0x15,
 		0x0a, 0x0d, 0x6f, 0x74, 0x68, 0x65, 0x72, 0x2f, 0x76, 0x65, 0x72, 0x73, 0x69, 0x6f, 0x6e, // apiversion
 		0x12, 0x04, 0x74, 0x65, 0x73, 0x74, // kind
-		0x12, 0x07, 0x6b, 0x38, 0x73, 0x00, 0x01, 0x02, 0x03, // data
```

Comment:
i checked the Encode(...), it does look only one prefix was added before runtime.Unknown, i.e., the runtime.Unknown.Raw does not have the prefix. It is hard to understand why this specific test was added to ensure runtime.Unknown.Raw have the prefix. But i think you're right.